.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,21 @@
+name: npm-publish
+on:
+  push:
+    branches:
+      - master # Change this to your default branch
+jobs:
+  npm-publish:
+    name: npm-publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+      - name: Set up Node.js
+        uses: actions/setup-node@master
+        with:
+          node-version: 10.0.0
+      - name: Publish if version has been updated
+        uses: pascalgn/npm-publish-action@51fdb4531e99aac1873764ef7271af448dc42ab4
+        env: # More info about the environment variables in the README
```

Comment:
In the repo readme? I don't see anything related to that there.

Reply:
I think the GitHub Actions author means his own Readme 🙂